.ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -21,9 +22,10 @@ pushd $local_path
 
 # Only skip tests if we can tell for sure that no go files were changed
 echo "Checking for modified go files"
+git fetch origin $old_branch:$old_branch --depth 1
```

Comment:
Is this redundant with L20 where we fetched new_branch with depth 2, since old branch's tip will always be the parent of new branch's latest commit?

Reply:
Either the parent or the latest commit itself (it wouldn't generate a new commit to the new_branch when there's no provider changes, which means the new and old branch have the same commit history for example: [auto-pr-9095](https://github.com/modular-magician/terraform-provider-google/commits/auto-pr-9095), [auto-pr-9095-old](https://github.com/modular-magician/terraform-provider-google/commits/auto-pr-9095-old)).
With the current change, we can have L20 to fetch the new branch with depth 1.
Does it answer your question?

Reply:
Yes- I think that that makes this line redundant in that case, if L20 already fetches this commit in all cases?

Reply:
ah, I see what you mean, I think L20 only fetch a single branch `new_branch`, so we wouldn't get other branches, at least based on my testing.

Reply:
Spoke offline- the commit is available based on the earlier clone, the issue is that we don't know that that commit is the commit referenced by the old branch's branch name.